docset/winserver2025-ps/smbshare/Revoke-SmbClientAccessToServer.md

```diff
@@ -0,0 +1,258 @@
+---
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: SmbClientAccessToServer.cdxml-help.xml
+Module Name: SmbShare
+ms.date: 02/22/2024
+online version: https://learn.microsoft.com/powershell/module/smbshare/revoke-smbclientaccesstoserver?view=windowsserver2025-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Revoke-SmbClientAccessToServer
+---
+
+# Revoke-SmbClientAccessToServer
+
+## SYNOPSIS
+Revokes SMB client access to a specified server.
+
+## SYNTAX
+
+```
+Revoke-SmbClientAccessToServer [-Name] <String[]> -IdentifierType <IdentifierType>
+ -Identifier <String> [-Force] [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+```
+Revoke-SmbClientAccessToServer -InputObject <CimInstance[]> -IdentifierType <IdentifierType>
+ -Identifier <String> [-Force] [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Revoke-SmbClientAccessToServer` cmdlet is used to remove a previously granted access rule for
+SMB clients to a specified server. When you run this cmdlet, it'll remove the firewall rule that
+allows incoming SMB traffic from the specified client IP address. If you need to grant access to
+the server again in the future, you can use the `Grant-SmbClientAccessToServer` cmdlet.
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+Revoke-SmbClientAccessToServer -Name "Server01" -IdentifierType "SHA256" -Identifier "ClientHash"
+```
+
+This example revokes the SMB client access to a server named "Server01" for a client with a
+specific SHA256 hash value.
+
+## PARAMETERS
+
+### -AsJob
+
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to
+complete.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CimSession
+
+Runs the cmdlet in a remote session or on a remote computer. Enter a computer name or a session
+object, such as the output of a [New-CimSession](/powershell/module/cimcmdlets/new-cimsession) or
+[Get-CimSession](https://go.microsoft.com/fwlink/p/?LinkId=227966) cmdlet. The default is the
+current session on the local computer.
+
+```yaml
+Type: CimSession[]
+Parameter Sets: (All)
+Aliases: Session
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Confirm
+
+Prompts you for confirmation before running the cmdlet.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: cf
```

Comment:
**Confirm** and **WhatIf** parameters go at the end of the list before CommonParameters.